source/persistency/PetaloPersistencyManager.cc

```diff
@@ -242,7 +241,7 @@ void PetaloPersistencyManager::StoreHits(G4HCofThisEvent* hce)
 	StoreIonizationHits(hits);
       }
     }
-    else if (hcname == SensorSD::GetCollectionUniqueName())
+    else if (hcname == ToFSD::GetCollectionUniqueName())
```

Comment:
In this case you are using `ToFSD` instead of `SensorSD` because we want to get the real time of the photons and not the bin so that we don't save the charge but each single photon, right?

Reply:
Well, actually that was a mistake, it should have been `ToFSD` from the start. It worked anyway because ToFSD used `SensorHits` before, like in nexus. However, now we've created a new class, so it doesn't work anymore.